oci/oci-how-to/deploy-pro-container-on-pro-kubernetes-cluster.rst

```diff
@@ -0,0 +1,202 @@
+The Kubernetes cluster needs to be on Ubuntu Pro nodes. Read more on the
+`Ubuntu Pro <https://ubuntu.com/pro>`_ website.
+
+
+Prerequisites
+-------------
+
+You will need
+
+- an Ubuntu Pro container image on a private registry. See how to build one
+  :doc:`here <build-ubuntu-pro-container-image>`.
+  The image should be kept private to avoid sharing your Ubuntu Pro subscription with
+  unwanted users since the image is linked to your Ubuntu Pro subscription.
+  
+  You can publish the image to your private registry. First, tag the image with your
+  private registry details using:
+
+  ::
+
+		docker tag <pro-image-name> <private-registry-pro-image-name>
+	
+  Finally, publish the image with:
+  
+  ::
+
+		docker push <private-registry-pro-image-name>
+
+- a Kubernetes cluster with a Ubuntu Pro subscription. See the 
+  :ref:`section below <deploy-pro-kubernetes-cluster>` to learn how to deploy it
+  for various clouds.
+
+
+.. _deploy-pro-kubernetes-cluster:
+
+Deploy Pro Kubernetes clusters
+------------------------------
+
+.. tabs::
+
+	.. tab:: EKS
+
+		Please see :doc:`here </aws/eks/eks-how-to/deploy-ubuntu-pro-cluster.rst>` to learn
+		how to deploy an Ubuntu Pro Kubernetes cluster on Elastic Kubernetes Service (EKS).
+
+	.. tab:: GCE
+
+		Please see :doc:`here </google/gce/gce-how-to/deploy-kubernetes-with-ubuntu-pro>` to learn
+		how to deploy an Ubuntu Pro Kubernetes cluster on Google Compute Engine (GCE).
+
+	.. tab:: OpenShift
+
+		See the `OpenShift guide <https://docs.openshift.com/container-platform/4.8/installing/index.html>`_
+		to learn how to create a cluster.
+
+
+		**Limitations**
+
+		OpenShift does not have Ubuntu Pro offerings for the nodes, i.e. you cannot
+		choose Ubuntu Pro images for the nodes.
+
+			"RHCOS is the only supported operating system for OpenShift Container Platform control plane, or master, machines. While RHCOS is the default operating system for all cluster machines, you can create compute machines, which are also known as worker machines, that use RHEL as their operating system."
+
+			-- `OpenShift docs <https://docs.openshift.com/container-platform/4.8/architecture/architecture-rhcos.html#rhcos-about_architecture-rhcos>`_
+
+
+		**Get an Ubuntu Pro subscription**
+
+		You can reach out `here <https://ubuntu.com/support/contact-us?product=contextual-footer-ua>`_
+		to attach the cluster nodes to an Ubuntu Pro subscription, or contact
+		`rocks@canonical.com <mailto:rocks@canonical.com>`_ if you need additional support.
+
+
+		**Success**
+
+		After attaching your Ubuntu Pro subscription to the cluster nodes, you will have an
+		Ubuntu Pro Kubernetes cluster running on OpenShift.
+
+	.. tab:: Tanzu
+
+		See the `Tanzu guide <https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/index.html>`_
+		to learn how to create a cluster.
+
+		**Get an Ubuntu Pro subscription**
+
+		You can reach out `here <https://ubuntu.com/support/contact-us?product=contextual-footer-ua>`_
```

Comment:
```suggestion
		If the resulting Kubernetes cluster is composed of Ubuntu nodes, you must attach them
		to a Pro subscription as described in `here <https://ubuntu.com/pro/tutorial>`_. Otherwise,
		if the Kubernetes cluster is not running on Ubuntu nodes, you can reach out 
		`here <https://ubuntu.com/support/contact-us?product=contextual-footer-ua>`_
```